VideoLocker/res/values/styles.xml

```diff
@@ -113,6 +113,15 @@
         <item name="android:background">@color/edx_brand_secondary_base</item>
     </style>
 
+    <style name="hairline_separator_style">
```

Comment:
Do we need the `_style` suffix here? The references will always have an `@style` prefix anyway… (Note that this applies to the previous base `separator_style` as well)